server/functions/framework/compiled_function.go

```diff
@@ -178,7 +178,9 @@ func (c *CompiledFunction) Eval(ctx *sql.Context, row sql.Row) (interface{}, err
 	resultTypes := c.callableFunc.GetParameters()
 	if len(c.casts) > 0 {
 		for i := range parameters {
-			if c.casts[i] != nil {
+			if parameters[i] == nil && c.callableFunc.GetIsStrict() {
+				return nil, nil
+			} else if c.casts[i] != nil {
```

Comment:
We should move this up to after we evaluate the parameters. We'll check `GetIsStrict`, and if it's true, then we'll do a loop over the parameters to find any nil values. Right now, we only do the check if we have casts, so we can end up sending a null through if the call doesn't require any casts.